### DISCUSSION TITLE
Can not find Pset from the selected bsDD

### DISCUSSION THREAD

**Discussion by ypleungaa** [IfcOpenShell/IfcOpenShell#5624] [General]
Hi, I have selected a bsDD and would like to use its Pset for assigning properties to a specific class. However, it seems that there are no Pset change found but I really can use its Pset before. I am using the bonsai version of 0.8.0. 

Please find the attached photos for more details. Thank you .

![image](https://github.com/user-attachments/assets/f67e5a17-7349-464e-9d35-f3d7bd5f78db)

**Comment by aothms** [comment]
I don't know enough about this, there seem to be indeed 3 properties for this entry in bsdd https://search.bsdd.buildingsmart.org/uri/wings/ZCP_LOD-I_300_AR/1.0.0/class/WIN
